index.html

```diff
@@ -1,10 +1,2413 @@
+                    <a href="#">
+                      As Judge Halts Census Wind Down, These States Have Lowest
+                      Response Rates
+                    </a>
+                  </h2>
+                  <p>
+                    Southeastern states Alabama and Georgia have the lowest
+                    response rates so far for the 2020 census.
+                  </p>
+                </div>
+              </article>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-sm-12 col-lg-12">
+
+          <div class="row flex-lg-row subscribe-block">
+            <div class="col-lg-7">
+              <div class="subscribe position-relative pt-3 pb-2 mt-3 mb-2 p-md-0 m-md-0 pt-lg-1">
+                <h6 class="text-red font-weight-bold text-uppercase pt-4 py-lg-2 pt-md-3 pb-md-1 m-lg-0">
+                  subscribe
+                </h6>
+                <div
+                  class="bg-light p-3 p-md-4 mb-4 mb-lg-0 p-lg-5 d-flex flex-column flex-md-row justify-content-center">
+                  <div class="d-flex mx-auto justify-content-center position-relative mb-2 pb-md-2 mb-md-2">
+                    <div class="mag-position mr-4 pr-1">
+                      <img class="ns-image"
+                        src="https://d.newsweek.com/en/full/1631397/cover-september-11-18-2020.jpg?w=170&q=95&f=87dabb57a32fecfaface1c7cf7a07d1f"
+                        alt="magazine thumb preview" />
+                      <img class="special-image"
+                        src="https://d.newsweek.com/en/full/1628669/cover-september-04-2020.jpg?w=170&q=95&f=1360030f30130d3440859fd318762628"
+                        alt="magazine thumb preview" />
+                    </div>
+                  </div>
+
+                  <div
+                    class="d-flex flex-column mx-auto align-items-center align-items-md-start align-items-lg-center justify-content-md-center ml-md-4 pl-md-0">
+                    <h5 class="subscribe-text font-weight-bold mt-4 text-wrap ">
+                      In-depth Stories You Don't See Elsewhere About Topics
+                      You Want to Read!
+                    </h5>
+                    <p class="subscribe-p text-secondary">All Access + Weekly Delivery.</p>
+                    <button type="button"
+                      class="btn-text bg-red border-0 btn-sm py-2 my-2 text-white text-center font-weight-bold text-uppercase">
+                      subscribe &nbsp; <i class="fas fa-chevron-right"></i>
+                    </button>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="col-lg-5">
+              <div class="newsletter position-relative mt-2 pt-3 pt-md-1 pt-lg-1 pb-lg-0 m-lg-0 border-bottom">
+
+                <h6 class="text-red font-weight-bold text-uppercase pb-1 mt-2 pt-3 pt-md-1 pt-md-2 p-lg-0">
+                  newsletter
```

Comment:
- [ ] Photo is missing under newsletter as present in original page
![chrome-capture (1)](https://user-images.githubusercontent.com/58504655/93229491-13565e00-f790-11ea-8ed7-06936eefc444.jpg)